tests/cli/base.py

```diff
@@ -74,3 +81,160 @@ def command_test(self, arg, kwarg=None, kwarg2=False):
                 pass
         argv = 'mage.py test'
```

Comment:
Using 'mage.py' name here distracts, suggests it might carry some important information. I think it actually doesn't.